numpy/f2py/auxfuncs.py

```diff
@@ -3,8 +3,8 @@
 
 Auxiliary functions for f2py2e.
 
-Copyright 1999,2000 Pearu Peterson all rights reserved,
-Pearu Peterson <pearu@ioc.ee>
+Copyright 1999 - 2011 Pearu Peterson all rights reserved.
+Copyright 2011 - 2023 NumPy Developers.
```

Comment:
Maybe remove the years, they only add maintenance burden without really affecting the legal status of the statement (see the link above). So something like this, or even without the NumPy developers line (which is covered by the regular license).
```suggestion
Copyright Pearu Peterson all rights reserved.
Copyright NumPy Developers.
```

Reply:
I understand the argument in the link, but here it would be nonsensical to have both all rights reserved under one name and also have the other. The years are important in the sense that it shows a transition in the copyright holders. So I would probably prefer `1999-2011` for Pearu and `2012 onwards` or `2012--present` for NumPy

Reply:
IANAL. I think the whole change is not needed, but am open to accepting other views.

Reply:
Fair enough, it was a single commit I'm happy to revert, the rest of this really needs to be in ASAP, lots of confusion regarding the `meson` backend and `f2py`.

Reply:
Reverted for now, though I think its worth a discussion, opened https://github.com/numpy/numpy/issues/25137 for later.